app/views/items/update_metadata.html.erb

```diff
@@ -20,14 +20,14 @@
   <div>
     <p>
       <div class="nypl-text-area-with-label">
-        <label class="required">Barcode(s)</label><span class="nypl-required-field"> Required</span>
-        <%= text_area_tag :barcodes, @barcodes.join(', '), { class: 'required', rows: 50, cols: 75 } %>
+        <label class="required" id="barcode-list">Barcode(s)</label><span class="nypl-required-field"> Required</span>
+        <%= text_area_tag :barcodes, @barcodes.join(', '), { class: 'required', rows: 50, cols: 75, aria: { labelledby: 'barcode-list barcodes-status' } } %>
```

Comment:
Since we have two IDs here for aria-labelledby. I think accessibility technologies will read it as "Barcode(s). Enter a single barcode or multiple barcodes separated by a comma or return."

Is this what we want?

Reply:
yes, per willa. that's what she wants.

Reply:
Approved then!!